docs/execplans/1-4-6-model-routed-transactions-and-session-gating.md

```
@@ -133,11 +133,13 @@ directories:
 
 The Session struct tracks per-connection authentication state:
 
-    pub struct Session {
-        pub user_id: Option<i32>,           // None = unauthenticated
-        pub privileges: Privileges,          // 38-bit bitmap
-        pub connection_flags: ConnectionFlags,
-    }
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)